README.md

```diff
@@ -1 +1,14 @@
-# tiny-builder
\ No newline at end of file
+# Paketo Tiny Builder
+
+This builder uses a Tiny base image (bionic build image, distroless run image)
```

Comment:
I think we might wanna say "distroless-like" as we do in our other documentation because "distroless" refers specifically to the google distroless container image.